features/facebookImport/src/views/explore/explore.jsx

```diff
@@ -34,10 +31,8 @@ const ExploreView = () => {
 
     const renderReportResult = () =>
         reportResult !== null && (
-            <PopUpMessage
-                reportResultAnswer={
-                    reportResult ? " successfully" : " unsuccessfully"
-                }
+            <PopUpBanner
+                reportType={reportResult ? "successful" : "unsuccessful"}
```

Comment:
If you export `types` you can use that instead of those literals here, having a single source of truth for these classes.